Templates/colors.json

```diff
@@ -0,0 +1,4687 @@
+        { "name": "Unknown209", "type": "uint32" },
+        { "name": "Unknown210", "type": "uint32" },
+        { "name": "Unknown211", "type": "uint32" },
+        { "name": "Unknown212", "type": "uint32" },
+        { "name": "Unknown213", "type": "uint32" },
+        { "name": "Unknown214", "type": "uint32" },
+        { "name": "Unknown215", "type": "uint32" },
+        { "name": "Unknown216", "type": "uint32" },
+        { "name": "Unknown217", "type": "uint32" },
+        { "name": "Unknown218", "type": "uint32" },
+        { "name": "Unknown219", "type": "uint32" },
+        { "name": "Unknown220", "type": "uint32" },
+        { "name": "Unknown221", "type": "float32" },
+        { "name": "Unknown222", "type": "float32" },
+        { "name": "Unknown223", "type": "float32" },
+        { "name": "Unknown224", "type": "float32" },
+        { "name": "Unknown225", "type": "float32" },
+        { "name": "Unknown226", "type": "uint32" },
+        { "name": "Unknown227", "type": "float32" },
+        { "name": "Unknown228", "type": "uint32" },
+        { "name": "Unknown229", "type": "uint32" },
+        { "name": "Unknown230", "type": "float32" },
+        { "name": "Unknown231", "type": "float32" },
+        { "name": "Unknown232", "type": "float32" },
+        { "name": "Unknown233", "type": "uint32" },
+        { "name": "Unknown234", "type": "float32" },
+        { "name": "Unknown235", "type": "uint32" },
+        { "name": "Unknown236", "type": "uint32" },
+        { "name": "Unknown237", "type": "float32" },
+        { "name": "Unknown238", "type": "float32" },
+        { "name": "Unknown239", "type": "uint32" },
+        { "name": "Unknown240", "type": "float32" },
+        { "name": "Unknown241", "type": "float32" },
+        { "name": "Unknown242", "type": "float32" },
+        { "name": "Unknown243", "type": "float32" },
+        { "name": "Unknown244", "type": "float32" },
+        { "name": "Unknown245", "type": "float32" },
+        { "name": "Unknown246", "type": "float32" },
+        { "name": "Unknown247", "type": "float32" },
+        { "name": "Unknown248", "type": "uint32" },
+        { "name": "Unknown249", "type": "uint32" },
+        { "name": "Unknown250", "type": "float32" },
+        { "name": "Unknown251", "type": "float32" },
+        { "name": "Unknown252", "type": "float32" },
+        { "name": "Unknown253", "type": "float32" },
+        { "name": "Unknown254", "type": "uint32" },
+        { "name": "Unknown255", "type": "uint32" }
+      ]
+    },
+    "BossEggMobileSpawner": {
+      "fields": [
+        { "name": "Datum", "type": "BossEggMobileSpawner::Datum" },
+        { "name": "Altitude", "type": "float32" },
+        { "name": "Unknown1", "type": "uint8" },
+        { "name": "Unknown2", "type": "uint8" },
+        { "name": "AppearOffset1D", "type": "float32" },
+        { "name": "Speed", "type": "float32" },
+        { "name": "Unknown3", "type": "uint8" }
+      ]
+    },
```

Comment:
Parameters named base on in-game functionality and parameters found on other objects with similar functionality, primarily `enmBigChaser`. `Unknown1` and `Unknown2` couldn't be named, however its likely they are the same as on `enmBigChaser` as the parameters line up with it up to the `Speed parameter`. `Unknown3` couldn't be named, however it has its value set to `15` in the retail game. None of the unknown parameters ever hit a data breakpoint in the retail game.